js/app/filters/importerrorfilter.js

```diff
@@ -29,17 +29,29 @@ app.filter('importErrorFilter', function () {
 		}
 
 		//TODO - use n instead of t to use proper plurals in all translations
-		switch(file.errors) {
-			case 0:
-				return t('calendar', 'Successfully imported');
-
-			case 1:
+		if (file.errors === 0) {
+			return t('calendar', 'Successfully imported');
```

Comment:
Is there a way we could bring in the number of imported events here ?

Reply:
Good call, will add it